.circleci/config.yml

```diff
@@ -0,0 +1,28 @@
+version: 2
+jobs:
+  build:
+    docker:
+      - image: regentmarkets/debian-pg:wheezy-5.26.2-96
```

Comment:
We'll need a public version of this image instead of this one; alternatively we build off https://hub.docker.com/_/postgres and install D::C requirements to it as needed.

Note also that if we want to test on Redis we'd need to arrange both Postgres and Redis as docker services at the start of the build run, injecting configuration as needed to the test container.

Reply:
Hi @zakame, this image is a public image and it already have Redis and Postgres included, [Link](https://hub.docker.com/r/regentmarkets/debian-pg)

are there any further changes needed?

Reply:
Right, I was thinking of the other image :joy_cat:

Reply:
So it's fine, right?

Reply:
Yep, this one's fine :+1: